### PR TITLE
Updates for Steeltoe v4

### PR DESCRIPTION
**Pull request by TimHess**
- add deprecation notes
- fix page titles and meta descriptions
- more-centralized redirects
- lots of cleanup

### CODE REVIEW

File: src/Client/appsettings.json [hunk 1]
```diff
@@ -25,37 +22,19 @@
...
-      },
-      {
-        "date": "<span>May</span> 10 2021",
-        "title": "Steeltoe 3.1.0 Release Candidate 1 is now available",
-        "description": "Steeltoe 3.1.0 brings the new Steeltoe Stream feature built on our RabbitMQ messaging components to the .NET community. Take a look and give us your thoughts.",
-        "url": "https://github.com/SteeltoeOSS/Steeltoe/releases/tag/3.1.0-rc1"
-      },
-      {
-        "date": "<span>january</span> 14 2021",
+        "date": "<span>January</span> 14 2021",
```
Comment (bart-vmware) [inline src/Client/appsettings.json:25]: Do these calendar events show up anywhere on the website? (Other than a banner for the current one, based on the current time)

File: src/Client/Components/Pages/Doc.razor
[the remark this answers is not in the document]
Comment (bart-vmware) [reply src/Client/Components/Pages/Doc.razor:1]: It's a blazor-specific warning.

File: src/Client/appsettings.json (hunk 1 above)
Comment (TimHess) [reply src/Client/appsettings.json:25]: These show up on the home page

File: src/Client/Components/Layout/SiteFooter.razor [hunk 2]
```diff
@@ -1,173 +1,108 @@
...
+                                    #github-icon .cls-2 { fill: #000000; fill-rule: evenodd; }
+                                </style>
+                            </defs>
+                            <path class="cls-1" d="M38,0a38,38,0,1,0,38,38A38,38,0,0,0,38,0Z"></path>
+                            <path class="cls-2" d="M38,15.59A22.95,22.95,0,0,0,30.71,60.3c1.15.21,1.57-.5,1.57-1.11s0-2,0-3.9c-6.38,1.39-7.73-3.07-7.73-3.07A6.09,6.09,0,0,0,22,48.86c-2.09-1.42.15-1.39.15-1.39a4.81,4.81,0,0,1,3.52,2.36c2,3.5,5.37,2.49,6.67,1.91a4.87,4.87,0,0,1,1.46-3.07c-5.09-.58-10.45-2.55-10.45-11.34a8.84,8.84,0,0,1,2.36-6.15,8.29,8.29,0,0,1,.23-6.07s1.92-.62,6.3,2.35a21.82,21.82,0,0,1,11.49,0c4.38-3,6.3-2.35,6.3-2.35a8.29,8.29,0,0,1,.23,6.07,8.84,8.84,0,0,1,2.36,6.15c0,8.81-5.37,10.75-10.48,11.32a5.46,5.46,0,0,1,1.56,4.25c0,3.07,0,5.54,0,6.29s.42,1.33,1.58,1.1A22.94,22.94,0,0,0,38,15.59Z"></path>
+                        </svg>
+                    </Href>
+                </div>
+                <div class="col">
+                    <Href href="https://twitter.com/steeltoeoss" title="Steeltoe on Twitter" NewWindow="true">
```
[the remark this answers is not in the document]
Comment (TimHess) [reply src/Client/Components/Layout/SiteFooter.razor:83]: Probably, but I don't know how to do this correctly with SVG, and haven't had luck in the amount of time I was willing to put in on it right now... Added #152 for follow-up

File: src/Client/Components/Layout/SiteFooter.razor
[the remark this answers is not in the document]
Comment (TimHess) [reply src/Client/Components/Layout/SiteFooter.razor:1]: Try accessing the PR slot using https://cookie-test.steeltoe.io/ -- it works as expected on my machine with this hostname

File: src/Client/Components/App.razor [hunk 3]
```diff
@@ -1,31 +1,25 @@
...
-
     <meta property="og:type" content="website">
-    <meta property="og:title" content="Create cloud-native .NET microservices with Steeltoe." />
-    <meta property="og:url" content="https://www.steeltoe.io" />
-    <meta property="og:image" content="https://steeltoe.io/images/favicon.png" />
-    <meta property="og:description" content="Create .NET microservices for modern platforms with Steeltoe. It's a friend of Spring Cloud and can be used anywhere." />
-    <meta name="twitter:title" content="Create cloud-native .NET microservices with Steeltoe.">
-
+    <meta property="og:url" content="https://www.steeltoe.io"/>
+    <meta property="og:image" content="https://steeltoe.io/images/favicon.png"/>
```
[the remark this answers is not in the document]
Comment (TimHess) [reply src/Client/Components/App.razor:20]: I somewhat haphazardly removed some of these - sure, they could be useful, but things like the URL, title and description shouldn't be the same across the entire site. As it was, these were counter-productive, and a social share looked the same no matter what link you tried to share. I'm pushing an update that will improve the situation, but I don't have a fix for all of it (such as only one image ever being used and not meeting size/scale recommendations)

**Review by bart-vmware — DISMISSED** [review]
I took the liberty of pushing some minor tweaks and fixes. LGTM now.

**Review by bart-vmware — APPROVED** [review]

**Pull request merged**